graylog2-server/src/main/java/org/graylog2/rest/resources/search/RelativeSearchResource.java

```diff
@@ -188,11 +190,20 @@ public TermsResult termsRelative(
             @QueryParam("query") @NotEmpty String query,
             @ApiParam(name = "size", value = "Maximum number of terms to return", required = false) @QueryParam("size") int size,
             @ApiParam(name = "range", value = "Relative timeframe to search in. See search method description.", required = true) @QueryParam("range") int range,
-            @ApiParam(name = "filter", value = "Filter", required = false) @QueryParam("filter") String filter) {
+            @ApiParam(name = "filter", value = "Filter", required = false) @QueryParam("filter") String filter,
+            @ApiParam(name = "order", value = "Sorting (field:asc / field:desc)", required = false) @QueryParam("order") String order) {
         checkSearchPermission(filter, RestPermissions.SEARCHES_RELATIVE);
 
+        final Sorting.Direction sortOrder;
+
+        if (isNullOrEmpty(order)) {
```

Comment:
You could use [`buildSorting()`](https://github.com/Graylog2/graylog2-server/blob/2.2.3/graylog2-server/src/main/java/org/graylog2/rest/resources/search/SearchResource.java#L226-L237) here, also see [`Sorting`](https://github.com/Graylog2/graylog2-server/blob/2.2.3/graylog2-server/src/main/java/org/graylog2/indexer/searches/Sorting.java).